src/core.c/CompUnit/Repository/Installation.rakumod

```diff
@@ -337,30 +337,26 @@ sub MAIN(:$name, :$auth, :$ver, *@, *%) {
             my $repo-prefix = self!repo-prefix;
             my $*DISTRIBUTION = CompUnit::Repository::Distribution.new($dist, :repo(self), :$dist-id);
             my $*RESOURCES = Distribution::Resources.new(:repo(self), :$dist-id);
-            my %done;
 
             my $compiler-id = CompUnit::PrecompilationId.new-without-check($*RAKU.compiler.id);
             for %provides.sort {
                 my $id = CompUnit::PrecompilationId.new-without-check($_.value.values[0]<file>);
                 $precomp.store.delete($compiler-id, $id);
             }
 
-            for %provides.sort {
```

Comment:
By doing this in parallel we lose the deterministic file creation order. However, I'm not sure that necessarily goes against the spirit of reproducibility or not.